Assets/MixedRealityToolkit.SDK/Experimental/Features/Utilities/TapToPlace/HideTapToPlaceLabel.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using Microsoft.MixedReality.Toolkit.Experimental;
+using Microsoft.MixedReality.Toolkit.Experimental.Utilities;
+using Microsoft.MixedReality.Toolkit.Utilities.Solvers;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using UnityEngine;
+
+public class HideTapToPlaceLabel : MonoBehaviour
+{
+    // Start is called before the first frame update
```

Comment:
remove comments, add private to functions

Reply:
Removed